phoenix-core/src/main/java/org/apache/phoenix/trace/PhoenixMetricsSink.java

```diff
@@ -175,7 +176,9 @@ private void createTable(Connection conn, String table) throws SQLException {
                         TAG_COUNT + " smallint, " +
                         ANNOTATION_COUNT + " smallint" +
                         "  CONSTRAINT pk PRIMARY KEY (" + TRACE.columnName + ", "
-                        + PARENT.columnName + ", " + SPAN.columnName + "))\n";
+                        + PARENT.columnName + ", " + SPAN.columnName + "))\n" +
+                        PhoenixDatabaseMetaData.TRANSACTIONAL + "=" + Boolean.FALSE;
```

Comment:
This is only for testing purposes. We have a config parameter that can be set so that tables are transactional by default. If that's set, we still don't want these system tables created as transactional tables, so we explicitly set that to false.